### DISCUSSION TITLE
Type error while deploy on vercel/cloudflare/local

### DISCUSSION THREAD

**Discussion by luthebao** [wevm/wagmi#2038] [Q&A] [answered]
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/20950325/226410291-67f8aace-1817-4d62-99d1-9a91a7f85118.png">

How could I handle this error ? Thank you all
I'm using wagmi ^0.12.2

**Comment by tmm** [comment, marked as answer]
Make sure you are using TypeScript 4.7.4 or greater

https://wagmi.sh/react/typescript
